.travis.yml

```diff
@@ -79,6 +79,16 @@ _helm_deploy: &helm_deploy
     export release_name_acs=acs-${TRAVIS_BUILD_NUMBER}-${STAGE_SUFFIX}
     export values_file=helm/alfresco-content-services/values.yaml
 
+    if [[ ${TRAVIS_BUILD_STAGE_NAME} == "esc" ]]; then
+      export search="elasticsearch"
+      export cluster="false"
```

Comment:
Cluster is disabled because we have a concurrency issue on license initialization, so the second node never find a valid license and it starts in read-only mode.